projects/plugins/jetpack/extensions/blocks/dialogue/dialogue.php

```diff
@@ -48,7 +48,7 @@ function filter_content( $content ) {
 		return '';
 	}
 
-	return wp_kses(
+	return wp_kses_post(
 		$content,
 		array(
```

Comment:
You should be able to get rid of the second argument now, you don't need it with `wp_kses_post`.